app/helpers/Intersect.js

```diff
@@ -399,10 +399,28 @@ function notifyUserOfRisk() {
  * @param {*} url
  */
 async function retrieveUrlAsJson(url) {
-  let response = await fetch(url, {
-    method: 'GET',
-    headers: { 'Content-Type': 'application/json', gov_do_token: GOV_DO_TOKEN },
-  });
+  let GOV_DO_TOKEN = await getToken();
+
+  const responseFunc = () => {
```

Comment:
you can add the GOV_DO_TOKEN variable in here and pass a parameter to the function if you need it, so you can call in the if statement the function
```suggestion
  const responseFunc = async (validToken) => {
       const GOV_DO_TOKEN = await getToken(validToken);
```

Reply:
What? I didn't understand